src/Application/Providers/ServiceProvider.php

```diff
@@ -3,6 +3,7 @@
 namespace Igni\Application\Providers;
 
 use Psr\Container\ContainerInterface;
+use Igni\Container\ServiceLocator;
```

Comment:
Names should be sorted in the ascending manner, please move this line before `Psr\Container\ContainerInterface`.
Thanks :)

Reply:
Ok